packages/eslint-config-typescript/tools/print-available-rules.js

```diff
@@ -1,4 +1,4 @@
-const plugin = require('@typescript-eslint/eslint-plugin');
+const plugin = require('@typescript-eslint/eslint-plugin/dist');
```

Comment:
The new version of `eslint-plugin`'s `package.json` file no longer has `"main": "dist/index.js",` and explicitly declares `"type": "commonjs"`. I don't know why it changed, but now we need to specify `/dist` or we get a lint error about the module not being found.